Tests/swift-sdk-swift-tests/RequestCreatorTests.swift

```diff
@@ -29,6 +29,50 @@ class RequestCreatorTests: XCTestCase {
         TestUtils.validateMatch(keyPath: KeyPath(JsonKey.endUnreadMessageCount), value: inboxSession.endUnreadMessageCount, inDictionary: body)
     }
     
+    func testGetInAppMessagesRequestFailure() {
+        let auth = Auth(userId: nil, email: nil)
+        let requestCreator = RequestCreator(apiKey: apiKey, auth: auth)
+        
+        let failingRequest = requestCreator.createGetInAppMessagesRequest(1)
+        
+        if let _ = try? failingRequest.get() {
+            XCTFail("request succeeded despite userId and email being nil")
+        }
+    }
+    
+    func testGetInAppMessagesRequest() {
+        let inAppMessageRequestCount: NSNumber = 42
+        
+        let request = createRequestCreator().createGetInAppMessagesRequest(inAppMessageRequestCount)
```

Comment:
Suggestion:
`        guard case let .success(request) = createRequestCreator().createGetInAppMessagesRequest(2) else {
            XCTFail("Could not create getInAppMessagesRequest")
            return
        }
        guard case let .get(getRequest) = request, let args = getRequest.args else {
            XCTFail("could not create getRequest")
            return
        }
`

Reply:
Now you don't need line numbers 61-69. No need for try? etc.